lib/ui/components/captured-requests.js

```diff
@@ -94,15 +93,19 @@ export class CapturedRequests extends React.Component {
     return requestList.map((request) => {
       return (
         <div key={ request.id }
-             className={ this._requestRowClassName(request.id) }
              onClick={ this._selectRequest.bind(this, request)}>
-          <div>
-            { this._activeIndicator(request) }
-            <MethodLabel method={ request.method }/>
-            { this._requestDetailsFor(request) }
-          </div>
-
-          { this._actionButtonFor(request) }
+          <RequestRow requestId={ request.id }
```

Comment:
We decided it makes sense in it's as a component, e.g. selected row etc.
So we should move the Draggable outside the component or something.